code/modules/pool/components/swimming.dm

```diff
@@ -125,6 +125,8 @@
 /datum/component/swimming/proc/drown(mob/living/victim)
 	if(victim.losebreath < 1)
 		victim.losebreath += 1
+	if(victim.stat > CONSCIOUS)
```

Comment:
This needs to be >= if you want unconcious or dead people to not emote. Also this removes the victim being made unconcious while unconcious.

Reply:
Tried a different approach, could you take a look?